editoast/src/views/infra/mod.rs

```diff
@@ -594,6 +575,8 @@ pub mod tests {
     use actix_web::test::call_service;
     use actix_web::test::read_body_json;
     use actix_web::test::TestRequest;
+    use diesel::sql_query;
+    use diesel::sql_types::BigInt;
```

Comment:
We also want to get rid of diesel in the views' tests. But this can be done in another PR later :)